lib/pdf_fill/forms/formatters/base.rb

```diff
@@ -24,6 +24,24 @@ def format_facility_label(value)
               "#{facility.station_number} - #{facility.name}"
             end
           end
+
+          # Formats a SSN to be in the format of 123-45-6789
+          def format_ssn(value)
+            return if value.blank?
+
+            digits = value.to_s.gsub(/\D/, '')
+
+            "#{digits[0..2]}-#{digits[3..4]}-#{digits[5..8]}"
+          end
+
+          # Formats a phone number to be in the format of (123) 456-7890
+          def format_phone_number(value)
```

Comment:
@JoshingYou1 Do you want to add any validation around length?
`return value unless digits.length == 10`?

Reply:
In theory the frontend should not allow for anything other than 10 digits. If. we notice any issues, we can revisit this.